tools/packaging/common/envoy_bootstrap.json

```diff
@@ -245,7 +245,7 @@
               "endpoint": {
                 "address":{
                   "pipe": {
-                    "path": "{{ .config.ConfigPath }}/SDS"
+                    "path": "./var/run/secrets/workload-identity/socket"
```

Comment:
Do we need to parametrize this path or it will always be in the same place all the time ?
I know that in the README it says it will be a fixed path, just wondering if this could be parametrized to have this configured easily in any scenario it might come up, and not require a code change to actually change the path

Reply:
It will be a fixed path, so Envoy always tries to connect to a socket on that path to get its certificates, and istio-agent always detects the presence or starts its own SDS server on the same path. This was agreed upon on a follow-up discussion with the istio community.